packages/ui-react/src/containers/HeaderLanguageSwitcher/HeaderLanguageSwitcher.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useConfigStore } from '@jwp/ott-common/src/stores/ConfigStore';
+import { useUIStore } from '@jwp/ott-common/src/stores/UIStore';
+
+import LanguageMenu from '../../components/LanguageMenu/LanguageMenu';
+
+const HeaderLanguageSwitcher = () => {
```

Comment:
Should we make it to Header folder in components?

Reply:
This is a container (wrapping the `LanguageMenu` component)